### PR TITLE
Update m365bp-increase-protection.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9872

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/business-premium/m365bp-increase-protection.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ To assign preset security policies, follow these steps:
 
 2. Go to **Email & Collaboration** \> **Policies & Rules** \> **Threat policies** \> **Preset Security Policies** in the **Templated policies** section. (To go directly to the **Preset security policies** page, use <https://security.microsoft.com/presetSecurityPolicies>.)
 
-3. On the **Preset security policies** page, in either the **Standard protection** or **Strict protection** section, change the toggle from **Disabled** to **Enabled**, and then select **Manage**.
+3. On the **Preset security policies** page, in either the **Standard protection** or **Strict protection** section,  select **Manage Portection Settings**.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-increase-protection.md]:
```suggestion
3. On the **Preset security policies** page, in either the **Standard protection** or **Strict protection** section, select **Manage Protection Settings**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Review by denisebmsft — APPROVED** [review]
Thank you @JohanFreelancer9

**Pull request merged**